lib/improver/tests/utilities/test_mathematical_operations.py

```diff
@@ -168,53 +168,140 @@ def test_descending_coordinate_negative(self):
             cube.coord(coord_name).points, self.descending_height_points)
 
 
-class Test_process(IrisTest):
+class Test_prepare_for_integration(IrisTest):
 
-    """Test the process method."""
+    """Test the prepare_for_integration method."""
 
     def setUp(self):
         """Set up the cube."""
         self.height_points = np.array([5., 10., 20.])
         cube = set_up_height_cube(self.height_points)[:, 0, :, :, :]
-        data = np.ones(cube.shape)
-        data[1] = np.zeros(cube[1].shape)+2
-        data[2] = np.zeros(cube[2].shape)+3
+        data = np.zeros(cube.shape)
+        data[0] = np.ones(cube[0].shape, dtype=np.int32)
+        data[1] = np.full(cube[1].shape, 2, dtype=np.int32)
+        data[2] = np.full(cube[2].shape, 3, dtype=np.int32)
         data[0, :, 0, 0] = 6
         cube.data = data
         self.cube = cube
 
     def test_basic(self):
-        """Test that a cube with the points on the chosen coordinate are
-        in the expected order."""
+        """Test that the type of the returned value is as expected and the
+        expected number of items are returned."""
         coord_name = "height"
         direction = "negative"
         result = (
             Integration(
                 coord_name, direction_of_integration=direction
-                ).process(self.cube))
+                ).prepare_for_integration(self.cube))
+        self.assertIsInstance(result, tuple)
+        self.assertEqual(len(result), 3)
```

Comment:
I think this unittest should also check that each item in the tuple is an iris cube.

Reply:
Added checks for items in the tuple.